contents/blog/product-360.md

```diff
@@ -0,0 +1,126 @@
+---
+date: 2022-01-18
+title: I used to think product people aren't needed. I was wrong
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+categories: ["CEO diaries", "Inside PostHog", "Engineering"]
+author: ["james-hawkins"]
+featuredImage: ../images/blog/product-people.png
+featuredImageType: full
+---
+
+When PostHog was fresh out of YC last year, there were two things we focused on: (i) build and (ii) talk to customers. Hiring a product person felt like it would slow us down. Engineers on our team would speak to our users (also engineers) - surely they'd know what's best to prioritize? We didn't want our engineers to be "shielded" from our customers.
+
+By [PostHog's series B](/blog/15-million-series-b), our team showed me that I was totally wrong. Here's why that was the case, what we did about it, and the results.
+
+>James Hawkins is PostHog Co-Founder & CEO. Connect with him on [LinkedIn](https://www.linkedin.com/in/j-hawkins/) and follow him on [Twitter](https://www.linkedin.com/in/j-hawkins/).
+
+## Why I used to think this way
```

Comment:
This section feels a bit jarring.

You have the most context, so I expect the compression ratio to be high: You understand exactly what you're trying to say here, simply based on a few key words.

It's not the same for me. I get where you're getting to, but I'd much rather prefer a short example showing this in action.

Here's an inferred expansion which I assume you mean (but can't confirm because I'm writing it, not you :P  - excuse my formatting/grammar/writing, just a quick example):

"I'd previously worked on building enterprise products with an enterprise sales team. This usually meant we always had this list of features that sales needed to close new customers, prioritised by their relative usefulness. Our goal was to build as many as we could.

There were invisible-to-me tradeoffs being made here: We'd get lots of initial enterprise contracts, but all these features didn't fit together coherently, which meant all these users would find the product frustrating to use, eventually leading to churn. Needless to say, there was no word-of-mouth growth"

.. and I realise now I didn't write it via an example but just as a story using the same bullets. lol.

Reply:
I quite like the bullets.  They're good pacing and readability and easy to scan.

Reply:
Fair, just mentioning my preference, but I understand it can be a minority opinion / not optimised for our goals :)

Reply:
Reading it back now, I kind of see your point... it just needs some connective tissue so it flows a bit better.

```suggestion
## Why I used to think this way

My previous experience of product management was something like this:

* Here's a list of features the enterprise sales team wants
* How much pipeline does each of these features influence?
* _Builds as much of the above as possible whilst trying to manage existing customer requests_

This lead to lots of initial enterprise contracts, but negative outcomes in the long term like:

* A product users find frustrating to use
* No word of mouth growth
* Customer churn in the long term

These were all traps I wanted to avoid for PostHog, and lead me to favour hiring more engineering talent over product people.
```

Reply:
Reading it back, I see your point. I've suggested a couple of small edits just to help it flow a little better.